ptime.go

```diff
@@ -1250,3 +1250,11 @@ func getWeekday(wd time.Weekday) Weekday {
 func (t *Time) resetWeekday() {
 	t.wday = getWeekday(t.Time().Weekday())
 }
+
+func (t Time) IsFuture() bool {
+	return t.Unix() > time.Now().Unix()
+}
+
+func (t Time) IsPast() bool {
```

Comment:
It is better to name the method [`Before`](https://pkg.go.dev/time#Time.Before) and use a compatible signature to ensure alignment with the standard `time` package.